bindings/csharp/csharp_code/PeerDASKZG.bindings/native_methods.g.cs

```diff
@@ -1,7 +1,4 @@
-//TODO: The class and methods in this file are currently public
-// We want to eventually make them internal.
-// This is blocked by csbindgen making all methods public.
-// See: https://github.com/Cysharp/csbindgen/pull/83
+using C = uint;
```

Comment:
Currently uint is changed to have type C, so I've added this in the header to alias C as uint, issue opened here: https://github.com/Cysharp/csbindgen/issues/89